projects/applicationinsights-angularplugin-js/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@microsoft/applicationinsights-angularplugin-js",
   "version": "2.9.1",
   "peerDependencies": {
-    "@angular/common": "^13.0.3",
-    "@angular/core": "^13.0.3"
+    "@angular/common": ">= 13.0.3",
```

Comment:
I think this upgrade should be backward compatible.
Maybe that the target needs to be reverted to `es2015` to be sure of this.

Reply:
Should consider Backward compatibility and the new solution which targets to es2020